### PR TITLE
Limit  character count for featured charm cards

### LINKED ISSUE #842: PostGRESQL summary text is partially hidden on card on homepage
Screenshot from 2021-02-26 09-35-59

### PR DESCRIPTION
**Pull request by steverydz**
## Done
Reduced character count on featured charm cards

## QA
- Go to https://charmhub-io-844.demos.haus/
- Compare the postgresql card to the one on the live homepage
- Check that the text is not too long and doesn't break the card

## Issue
Fixes #842

### CODE REVIEW

**Comment by solazio** [comment]
Can we truncate it with `...` at the end. Looks weird like this.

**Comment by steverydz** [comment]
> Can we truncate it with `...` at the end. Looks weird like this. You can use this library https://github.com/canonical-web-and-design/charmhub.io/blob/master/static/js/src/libs/truncate-string.js

@solazio done

File: templates/partial/_featured-charms.html [hunk 1]
```diff
@@ -14,7 +14,7 @@ <h3 class="p-card__title p-heading--4 u-no-margin--bottom package-card-title">{{
     </div>
 
     <div class="p-card__content">
-      <p><small class="package-card-summary">{{ charm.result.summary }}</small></p>
+      <p><small class="package-card-summary">{{ charm.result.summary[:90].strip() }}{% if charm.result.summary[:90]|length == 90 %}&hellip;{% endif %}</small></p>
```
Comment (solazio) [inline templates/partial/_featured-charms.html:17]: In the template you could just do:
```suggestion
      <p><small class="package-card-summary">{{ charm.result.summary|truncate(90) }}</small></p>
```

**Review by solazio — APPROVED** [review]
Just one small suggestion in the code.

**Pull request merged**